packages/stryker/src/initializer/StrykerInitializer.ts

```diff
@@ -0,0 +1,133 @@
+import * as child from 'child_process';
+import { StrykerInquirer } from './StrykerInquirer';
+import NpmClient from './NpmClient';
+import PromptOption from './PromptOption';
+import * as log4js from 'log4js';
+import { filterEmpty } from '../utils/objectUtils';
+import StrykerConfigWriter from './StrykerConfigWriter';
+
+const log = log4js.getLogger('StrykerInitializer');
+
+export default class StrykerInitializer {
+
+  private inquirer = new StrykerInquirer();
+
+  constructor(private out = console.log, private client: NpmClient = new NpmClient()) { }
+
+  /**
+   * Runs the initializer ask used framework and test runner en setup environment
```

Comment:
This comment will become obsolete if any of the internals change. `initialize()` prepares your project to be used in Stryker